src/main/resources/db/changelog/changes/17-create-order-item-table.yaml

```diff
@@ -0,0 +1,42 @@
+databaseChangeLog:
+  - changeSet:
+      id: create_order_item_table
+      author: mhorianin
+      changes:
+        - createTable:
+            tableName: order_item
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: order_id
+                  type: bigint
+                  constraints:
+                    nullable: false
+              - column:
+                  name: book_id
+                  type: bigint
+                  constraints:
+                    nullable: false
+              - column:
+                  name: quantity
+                  type: INTEGER
+                  constraints:
+                    nullable: false
+              - column:
+                  name: price
+                  type: decimal
+                  constraints:
+                    nullable: false
+        - addForeignKeyConstraint:
+            baseTableName: order_item
+            baseColumnNames: book_id
+            referencedTableName: books
+            referencedColumnNames: id
+            constraintName: fk_book_order_item
+            onDelete: CASCADE
```

Comment:
```suggestion
            onDelete: CASCADE
            
```

Reply:
I did not understand this comment. Should I delete this line?

Reply:
You need to add empty line at the end of file